It appends system name to library names for cross compile. For example, the
library name is libwx_baseu-3.1-Linux.so rather than libwx_baseu-3.1.so. It is
not appropriate for oe.

Upstream-Status: Pending [oe specific]

Signed-off-by: Kai Kang <kai.kang@windriver.com>
---
 build/cmake/functions.cmake | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/build/cmake/functions.cmake b/build/cmake/functions.cmake
index 53cd6a1ace..95efa17fe4 100644
--- a/build/cmake/functions.cmake
+++ b/build/cmake/functions.cmake
@@ -211,9 +211,9 @@ function(wx_set_target_properties target_name is_base)
         endif()
     else()
         set(cross_target "")
-        if (CMAKE_CROSSCOMPILING)
-            set(cross_target "-${CMAKE_SYSTEM_NAME}")
-        endif ()
+        #if (CMAKE_CROSSCOMPILING)
+        #    set(cross_target "-${CMAKE_SYSTEM_NAME}")
+        #endif ()
 
         set_target_properties(${target_name}
             PROPERTIES
